src/dbnode/storage/database.go

```diff
@@ -223,12 +224,39 @@ func NewDatabase(
 	return d, nil
 }
 
+func (d *db) UpdateSchemaRegistry(newNamespaces namespace.Map) {
+	schemaReg := d.opts.SchemaRegistry()
+	schemaUpdates := newNamespaces.Metadatas()
+	for _, metadata := range schemaUpdates {
+		curSchemaId := "none"
```

Comment:
nit: Go naming suggests this should be `curSchemaID`:
https://github.com/golang/lint/issues/124

Reply:
ditto

Reply:
Ping, think you still need to rename to `curSchemaID`